### PR TITLE
improved @babel/traverse docs

### PR DESCRIPTION
**Pull request by oyilmaztekin**
This is an improvement about @babel/traverse method. An alternative usage has been added to the doc.
[babel/website#1934](https://github.com/babel/website/issues/1934)

### CODE REVIEW

File: website/versioned_docs/version-7.0.0/traverse.md [hunk 1]
```diff
@@ -33,5 +33,17 @@ traverse(ast, {
   }
 });
 ```
+
+Also, we can target particular [**node types**](https://babeljs.io/docs/en/babel-types#api) in the Syntax Tree
+
+```js
+traverse(ast, {
+    FunctionDeclaration: function(path) {
+             path.node.name = "x";
```
Comment (suchipi) [inline website/versioned_docs/version-7.0.0/traverse.md]: FunctionDeclaration nodes don't have a `name` property; could you change this to Identifier or change the code?

Comment (oyilmaztekin) [reply website/versioned_docs/version-7.0.0/traverse.md]: Sorry. Id property of the FunctionDecleration node has a name.  I missed. I'm on it.

Comment (oyilmaztekin) [reply website/versioned_docs/version-7.0.0/traverse.md]: I fixed it. Could you please check...

File: website/versioned_docs/version-7.1.0/traverse.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (suchipi) [inline website/versioned_docs/version-7.1.0/traverse.md]: You don't need to add this; all versions of 7.x fall back to the 7.0 docs if there's no file in this directory

**Pull request merged**